recipes/paraprobe/meta.yaml

```diff
@@ -0,0 +1,93 @@
+{% set name = "paraprobe" %}
+{% set version = "v0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://gitlab.com/{{ name }}/{{ name }}-toolbox/-/archive/{{ version }}/{{ name }}-toolbox-{{ version }}.tar.gz
+    sha256: ded5193f54f6aa63fea3092f0cf8a680d2ad96501936e830ef1ba7f298595577
+    folder: paraprobe-toolbox
+    patches:
+      - paraprobe.patch
+  - fn: v0.4.6.tar.gz
+    url: https://github.com/chr1shr/voro/archive/refs/tags/v0.4.6.tar.gz
```

Comment:
This needs to be a separate package, even if it is header-only.

Reply:
The source code is shipped with this package. So I have changed it to use the shipped version instead of downloading the package.